gst/nnstreamer/elements/gsttensor_trainer.c

```diff
@@ -18,6 +18,10 @@
  * ]|
  *
  * Total number of data to be received is 1000((num-training-samples + num-validation-samples) * epochs)
+ * 
+ * output tensors : dimensions=(string)1:1:4:1:1:1:1:1, types=(string)float64.
```

Comment:
```dimensions=(string)1:1:4``` should be enough. Let's not verbose dimensions (we are expanding that to 16 soon)

Reply:
Thank you. I will remove it.